stl/inc/xloctime

```diff
@@ -554,26 +558,27 @@ private:
             }
         }
 
-        bool _Seendigit = false;
+        int _NSeendigit = 0;

```

Comment:
Similarly, `_Seen_digits` or better yet `_Digits_seen` would be more conventional/readable.